roles/edge-image-builder/templates/config-eib.yaml.j2

```diff
@@ -0,0 +1,8 @@
+apiVersion: 1.0
+image:
+  imageType: raw
+  baseImage: {{ os_image_name }}
+  outputImageName: {{ eib_output_image_name }}
+operatingSystem:
+  kernelArgs:
+  - ignition.platform.id=openstack
```

Comment:
I'd say to add a note on why is this needed as 'openstack' feels weird in this context :)

Reply:
I don't want to add nothing into the template. I will look for a place in the other manifests

Reply:
I will add it... good suggestion